client/src/app/pages/applications/components/application-assessment-status/application-assessment-status.tsx

```diff
@@ -28,8 +32,9 @@ const getStatusIconFrom = (assessment: Assessment): IconedStatusPreset => {
 
 export const ApplicationAssessmentStatus: React.FC<
   ApplicationAssessmentStatusProps
-> = ({ assessment, isLoading, fetchError }) => {
+> = ({ assessments, isLoading, fetchError }) => {
   const { t } = useTranslation();
+  const { assessment } = useFetchAssessmentByID(assessments?.[0]?.id || 0);
```

Comment:
Limiting the application to a the single first assessment is temporary?  

Will there be times where the status of multiple assessments for a single application will be available and shown?

If there are multiple, will the 0th assessment always be the most current one and that is the one to show the status on?

Reply:
That is a great question re: how are we showing net questionnaire status with multiple potential active questionnaires/assessments.

Reply:
I raised this with Justin & Jeff. Not sure if we can expect a hub status for applications / archetypes around status. Probably need to think about if we want to perform an operation to check if "some" of the assessments have an in progress status etc. I will open an issue around this.

Reply:
https://github.com/konveyor/tackle2-ui/issues/1324

Reply:
Looks like currently there is an "assessed" bool on the application within the hub struct. So maybe we just keep it simple for now and use that.